### ISSUE TITLE
Bug: NineSliceSpritePipe GpuSprite Undefined

### ISSUE THREAD

**Issue by NerdHorse** [pixijs/pixijs#11135]
### Current Behavior

I don´t know why this is happing
![Image](https://github.com/user-attachments/assets/a1f32500-38ea-4e0e-a6e7-99f2153cc41c)
![Image](https://github.com/user-attachments/assets/0290a6a6-c002-41e4-a7a1-9a9ad59a3204)

### Expected Behavior

Dont crash the render

### Steps to Reproduce

If I switch to another browser tab and return a few seconds later, the crash occurs.

### Environment

- **`pixi.js` version**: * 8.6.3*
- **Browser & Version**: * Chrome 131.0.6778.108*
- **OS & Version**: *Windows*
- preference: "webgpu"

**Comment by NerdHorse** [comment]
I conducted several experiments.
I suspected it might be an issue with the garbage collector.
First, I changed the preference to WebGL. The problem persisted.
Then, I set `renderer.textureGC.active = false`. The problem persisted.
After that, I set `renderer.renderableGC.enabled = false`. The problem stopped happening.

I kept stacking actions, so I'm not sure if simply setting `renderableGC` to `false` is enough. I'll check this tomorrow.

**Comment by GoodBoyDigital** [comment]
thanks for sharing this! I look at this one as a priority!

**Comment by NerdHorse** [comment]
For more details and context:
I created a popup (a container with several objects inside).
I added it to the screen, ran the entry animation, viewed the popup, ran the exit animation, and removed the popup from the screen.
Then, I switched to another tab, stayed out for 1–2 minutes, and came back.
Afterward, I clicked to open the popup again and follow the same flow, but the render crashes. The screen goes black, and that error is logged several times.

I noticed that switching tabs isn’t necessary. The issue occurs if I simply close the popup and wait for some time before opening it again.
